### PR TITLE
[SMALLFIX] Changed $bin variable to ${BIN} on all shell files

### PR DESCRIPTION
**Pull request by caadecarvalho**
This smallfix defines a standard for old $bin variable inside all bin/tachyon* files

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by caadecarvalho** [comment]
Hi @jsimsa , I opened this smallfix to address the standard `BIN` variable and also to correct the issue 'bash: line 4: /../libexec/tachyon-config.sh: No such file or directory'

File: bin/tachyon-mount.sh [hunk 1]
```diff
@@ -18,7 +17,7 @@ Usage="Usage: tachyon-mount.sh [Mount|SudoMount] [MACHINE]
   workers\t\tMount all the workers on worker nodes"
 
 function init_env() {
-  
+  BIN=$(cd "$( dirname "$1" )"; pwd)
```
Comment (jsimsa) [inline bin/tachyon-mount.sh]: why is this line needed? wasn't BIN already initialized on line 12?

**Comment by jsimsa** [comment]
@caadecarvalho thank you for the PR, left one comment

**Comment by caadecarvalho** [comment]
@jsimsa , thanks for the comments, I duplicated because I was receiving the error, after a better understanding, I realize that when under sudo elevation, `$BIN` variable wasn't set (as it was set on other user session).

**Comment by jsimsa** [comment]
LGTM ... please resolve merge conflicts

**Comment by caadecarvalho** [comment]
thanks, @jsimsa . Conflicts merged.

**Comment by jsimsa** [comment]
LGTM ++
